compiler/rustc_resolve/src/access_levels.rs

```diff
@@ -25,7 +22,7 @@ impl<'r, 'a> AccessLevelsVisitor<'r, 'a> {
     pub fn compute_access_levels<'c>(r: &'r mut Resolver<'a>, krate: &'c Crate) {
         let mut visitor = AccessLevelsVisitor { r, changed: false };
 
-        visitor.set_access_level_def_id(CRATE_DEF_ID, Some(AccessLevel::Public));
+        visitor.r.access_levels.try_init(CRATE_DEF_ID, None);
```

Comment:
```suggestion
        visitor.update(CRATE_DEF_ID, ...);
```
The parent id for `fn update` can be changed from `LocalDefId` to `Option<LocalDefId>` if necessary.